applications/luci-app-openwisp/htdocs/luci-static/resources/view/openwisp.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+'require view';
+'require form';
+'require tools.widgets as widgets';
+
+return view.extend({
+	render: function () {
+		var m, s, o;
+
+		m = new form.Map('openwisp', 
+			_('OpenWISP'), 
+			_("Configure, start and stop the OpenWISP agent on this device. Read more about configuration values: <a href='https://github.com/openwisp/openwisp-config'>https://github.com/openwisp/openwisp-config</a>"));
```

Comment:
Shouldn't it be opened in a new tab?

Reply:
I don't know what you mean, there are already two tabs: General and Advanced

Reply:
Well, there is a link for OpenWisp config and I wonder, if the URL should not be opened in a new tab.

Reply:
Ah, that makes sense.  Yeah let's do that (pushed)